core/designsystem/src/main/java/org/mifos/mobilewallet/mifospay/designsystem/component/MifosFaqItemHolder.kt

```diff
@@ -0,0 +1,86 @@
+package org.mifos.mobilewallet.mifospay.designsystem.component
+
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.animation.core.Spring
+import androidx.compose.animation.core.spring
+import androidx.compose.animation.expandVertically
+import androidx.compose.animation.fadeIn
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowDropDown
+import androidx.compose.material3.Divider
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.scale
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun MifosFaqItemHolder(
+    question: String?,
+    answer: String?,
+    isSelected: Boolean,
+    onItemSelected: (Int) -> Unit,
+    index: Int
+) {
+    Column(
```

Comment:
make a `remember` variable of `isSelected` and make it true false on item click and pass it to below AnimatedVisibility

Don't need to pass the isSelected from data source it can be manage here.